clang/test/Driver/unsupported-option-gpu.c

```diff
@@ -0,0 +1,7 @@
+/// Some target-specific options are ignored for GPU, so %clang exits with code 0.
+// DEFINE: %{gpu_opts} = --cuda-gpu-arch=sm_60 --cuda-path=%S/Inputs/CUDA/usr/local/cuda --no-cuda-version-check
```

Comment:
these defines seem overkill and harder to read compared to just duplicating the command line twice

Reply:
I think `gpu_opts` can be merged into `%{check}`. `%{check}` should be kept as it will become longer soon.

Reply:
For the purpose of warning checking, we can drop this altogether and use `-nogpuinc -nogpulib` . Whatever default GPU we end up targeting is also irrelevant.